scripts/index.js

```diff
@@ -34,7 +34,7 @@ async function run() {
   }
   issues = filterIssuesBylabels(issues, labels);
   issues = getData(issues);
-  return saveMultipleLogs(issues);
+  return saveMultipleLogs(moment().year(), issues);
```

Comment:
This doesn't work properly if your last update was mid-december and your updating in January. Issues from the end of december will be in the wrong year.